CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## [Unreleased]
+## [0.23.0]
```

Comment:
We need a `Full AMD support` in `Added` section.

Reply:
Done